doc/user/spec_examples.md

```diff
@@ -0,0 +1,124 @@
+# Cluster Spec Examples
+
+### Three members cluster
+
+```yaml
+spec:
+  size: 3
+```
+
+This will use default version that etcd operator chooses.
+
+### Three members cluster with version specified
+
+```yaml
+spec:
+  size: 3
+  version: 3.1.1
+```
+
+### Three members cluster with node selector and anti-affinity
+
+```yaml
+spec:
+  size: 3
+  pod:
+    nodeSelector:
+      diskType: ssd
+    antiAffinity: true
+```
+
+### Three members cluster with resource requirement
+
+```yaml
+spec:
+  size: 3
+  pod:
+    resources:
+      limits:
+        cpu: 300m
+        memory: 200Mi
+      requests:
+        cpu: 200m
+        memory: 100Mi
+```
+
+### Three members cluster with PV backup
+
+See [example](../../example/example-etcd-cluster-with-backup.yaml) .
+
+### Three members cluster with S3 backup
+
+```yaml
+spec:
+  size: 3
+  version: "3.1.0"
+  backup:
+    backupIntervalInSecond: 300
+    maxBackups: 5
+    storageType: "S3"
+```
+
+### Three members cluster that restores from previous PV backup
+
+If we created a cluster "cluster-a" and it had backup, but then it was deleted,
```

Comment:
If a cluster `cluster-a` was created with backup, but deleted or failed later on, we can recover the data as long
as the PV still exists.

The following spec is an example to do so: